templates/lxc-download.in

```diff
@@ -48,17 +46,17 @@ DOWNLOAD_USE_CACHE="false"
 DOWNLOAD_VALIDATE="true"
 DOWNLOAD_VARIANT="default"
 
+DOWNLOAD_KEYID="0xE7FB0CAEC8173D669066514CBAEFF88C22F6E216"
+if [[ -z "$DOWNLOAD_KEYSERVER" ]]; then
+    DOWNLOAD_KEYSERVER="hkp://pool.sks-keyservers.net:80"
```

Comment:
Does all of the sks pool now support port 80? it used to be that only p80.pool.sks-keyservers.net supported it.